asv/benchmarks.py

```diff
@@ -9,8 +9,6 @@
 import re
 import tempfile
 import itertools
-import datetime
```

Comment:
Is this file already taken care of in the other PR? Not sure if I'm getting confused, or if there is some reasoning I'm not understanding.

Reply:
THe other is asv/benchmark.py but I will merge them all to this PR to avoid confusion